docker/BUILD

```diff
@@ -48,3 +61,12 @@ docker_build(
         ":proxy",
     ],
 )
+
+docker_build(
+    name = "app",
+    base = "@docker_ubuntu//:xenial",
+    entrypoint = ["/bin/server"],
```

Comment:
I'm building on xenial, so it was a path of least resistance in matching Envoy GLIBC versions.
We can spend effort in trying out alpine. It's not a big deal to use larger base as long as we reuse the base layer in docker.